### DISCUSSION TITLE
Higher k's and possible mainnet attacks

### DISCUSSION THREAD

**Discussion by delucca** [madMAx43v3r/chia-plotter#153] [General]
Hi everyone :)

I don't know if you're all following the discussions over the Reddit community, but some users are reporting ~25 minutes plotting time.

That led to a concern regarding the safety of Chia mainnet. It seems that Chia is vulnerable to "plot on-demand" attacks. To be fair, I've not read the entire green paper yet, so I am not 100% sure that an attack like that can happens, but some users are stating that if we can plot in less than 30 seconds then an attack like that is feasible.

In any case, I would like to start the discussion regarding implementing higher k's in this plotter, for the following reasons:

1. As I've stated above, it is possible for Chia to deprecate k32 plots, since some users are achieving such low plotting times
2. Even so, some users bought large NVMe (1TB +) to parallel plot, and this plotter works best in serial, since it optimizes resource usage. So, some users (like me) are wasting resources since we could be plotting k35 instead of k32.

**Comment by madMAx43v3r** [comment]
Check this discussion over here: https://github.com/Chia-Network/chia-blockchain/discussions/452

**Reply by delucca** [reply]
Glad to know, I'll check there! Thanks.

**Comment by Hardcore-fs** [comment]
LOL... I did say this less than a 2 weeks ago.
I  don't think some of the people involved are clear on the sort of hardware  out there.....
There are people working in computer shops, who just take the stock & build rigs.
then spend their days mining crypo coin.
Others are working in Data centers......

**Reply by delucca** [reply]
In any case, Chia maintainers have already explained that Chia is no longer vulnerable to "fast plotting" attacks. So, no problem with that!

**Comment by Hardcore-fs** [comment]
go read all the research.....to put fast plotting in context
